rfc/php_technical_committee.wiki

```diff
@@ -52,83 +51,121 @@ For example if there is a proposal to add accessors for PHP classes, then everyt
-problem and put label on it.
+* Administering the election of the next set of TC members.
+* Documenting and communicating any decisions of the TC.
+* In case of a tie, the Secretary's vote will be considered the winning position.
 
-The TC decision should not take longer than month. If it takes longer than one month, the following work flow can be
-applied:
-  - After a month of no response from the TC the core developer can send a formal notification to the internals mailing list.
-  - If the notification is ignored for more than one month by the TC, the core developer can send a formal warning to the internals mailing list.
-  - If the formal warning is also ignored for more than a month, then the core developer can call for new election of TC members
-  - New election happen within a month.
+=== Elections ===
 
-There is a caveat that if the election happens due to yearly elections between any steps of the workflow above, the
-core developer needs to send a new notification after the elections and starts the workflow again.
+The election process takes place annually, concurrent with the election of a release manager in approximately
+March/April.  (The exact date may vary year to hear.)  The sitting TC Secretary officially begins the process with
+a call for nominations.  The nomination period lasts for one week.
 
-==== TC membership ====
+During that week, any eligible core developer may self-nominate by posting to the Internals mailing list.  If the
+candidate is paid to work on php-src or PHP extensions, they must state in their nomination which company or organization
+they work for.  Failure to do so will result in disqualification, as determined by the TC Secretary.
+
+Once nominations are over, the voting process will begin.  Eligible voters are the same as for any RFC.
+
+If there are five or fewer nominees, there is no vote and all nominated individuals are elected.
+
+If there are six or more nominees, a two week vote using the Single Transferable Vote method will take place.  The five
+winning candidates at the end of the vote are elected.
+
+The newly elected TC members will elect from their own number a TC Secretary by simple majority.
+
+=== Vacancies ===
+
+Any TC member may resign at any time via an email to the Internals mailing list.  If the resigning member is the TC
+Secretary, then the TC will elect a new Secretary from its number.
 
-The TC candidates nominate themself to become a TC member. They must be a core developers and should have a general
-knowledge about PHP internals. If the candidates are paid to work on php-src or PHP extensions, they must state in
-their nomination which company or organization they work for.
+If the number of TC members is reduced to three or fewer and there are more than two months until the next regularly
+scheduled election, the Secretary must call for a special election for the vacant seats.  The election process is the
```

Comment:
There should be probably some provision in case all members decide to resign so then it should be probably the same like when it is considered AWOL as described below - it means release manager should step in.

Reply:
Hm, perhaps we move the RM mention up here then?

Reply:
Yeah good idea.

Reply:
Done.